src/entities/Place/model.ts

```diff
@@ -558,6 +558,35 @@ export default class PlaceModel extends Model<PlaceAttributes> {
     return results[0].total
   }
 
+  static async findWorldNames(): Promise<{ world_name: string }[]> {
+    const sql = SQL`
+      SELECT p.world_name
+      FROM ${table(this)} p
+      WHERE
+        p.disabled is false AND world is true
+      ORDER BY p.world_name ASC
+    `
+
+    return await this.namedQuery("find_world_names", sql)
+  }
+
+  static async countWorldNames() {
+    const query = SQL`
+      SELECT
+        count(*) as total
+      FROM ${table(this)} p
+      WHERE
+        p.disabled is false
+        AND p.world is true
+    `
+    const results: { total: number }[] = await this.namedQuery(
+      "count_world_names",
+      query
+    )
+
+    return results[0].total
```

Comment:
parse `total` to Number: `return Number(results[0].total)`. sql `count` returns a string in JS